frontend/add-client/form-inputs/demographic-information-inputs.component.tsx

```diff
@@ -126,7 +125,6 @@ export default function DemographicInformationInputs(
             type="number"
             value={juvenileDependents}
             onChange={evt => setJuvenileDependents(Number(evt.target.value))}
-            required
```

Comment:
Why are we making Juvenile Dependents optional? It doesn't seem related at all to date of US arrival.